source/docs/software/pathplanning/index.rst

```diff
@@ -3,6 +3,16 @@ Path Planning
 
 Path Planning is the process of creating and following trajectories. These paths use the WPILib trajectory APIs for generation and a :ref:`Ramsete Controller <docs/software/advanced-controls/trajectories/ramsete:Ramsete Controller>` for following. This section highlights the process of characterizing your robot for system identification, trajectory following and usage of PathWeaver. Users may also want to read the :ref:`generic trajectory following documents <docs/software/advanced-controls/trajectories/index:Trajectory Generation and Following with WPILib>` for additional information about the API and non-commandbased usage.
 
+Notice on Swerve Support
+------------------------
+
+Swerve support in path following has a couple of hiccups that teams need ot be aware of:
+- WPILib currently does not support swerve in simulation, please see `this <https://github.com/wpilibsuite/allwpilib/pull/3374>`__ pull request.
+- SysID only supports tuning the swerve heading using a simple motor project and is unable to collect module velocity data.
```

Comment:
"SysID only supports tuning the swerve heading"
Is heading the best word to use here? I assume you mean the swerve azimuth or tuning the swerve turning motor. Also, is a simple motor project still a thing in SysID? To my knowledge, it is just general mechanism now.